home-work-app/src/app/redux/actions/movieActionsAdapter.ts

```diff
@@ -0,0 +1,28 @@
+import { IApiResponse } from '../../interfaces/IApiResponse';
+import { MovieService } from '../../services/MovieService';
+import { store } from '../store';
+
+export const actionAdapter: any = ({movieId, command, formData}: any): any => (dispatch: any) => {
+    const queryParams: any = store.getState().movieEditor.queryParams;
+
+    MovieService.movieActionRequest({id: movieId, command, formData})
```

Comment:
This `movies/{id}` request you need only for deleting and getting video description.
You need to create 3 separate async action creators (addMovieAction, editMovieAction and deleteMovieAction) with different logic inside.
All this actions will dispatch action, that set movies list (SET_MOVIES_LIST, for example), after getting response from `get` movies request.